security_monkey/auditor.py

```diff
@@ -165,14 +167,17 @@ def _is_current_method_disabled(self):
 
         return False
 
-
     def audit_all_objects(self):
         """
         Read all items from the database and inspect them all.
         """
         self.items = self.read_previous_items()
         self.audit_these_objects(self.items)
 
+    def audit_specific_items(self, items):
```

Comment:
The names `audit_specific_items` and `audit_these_objects` seem to mean the same thing.

Why not delete `audit_specific_items` and have `audit_these_objects` always set `self.items = items`.